traffic.go

```diff
@@ -1,85 +1,147 @@
 package main
 
 import (
-	"image"
 	"image/color"
 
 	"engo.io/ecs"
 	"engo.io/engo"
-	"github.com/EngoEngine/TrafficManager/systems"
+	"engo.io/engo/common"
+	"github.com/EngoEngine/TrafficDefense/systems"
```

Comment:
Ok I realized you changed TrafficManager to TrafficDefense because the game changed.